cmd/sync.go

```diff
@@ -313,8 +313,14 @@ func doSync(c *cli.Context) error {
 	go func() { _ = http.ListenAndServe(fmt.Sprintf("127.0.0.1:%d", config.HTTPPort), nil) }()
 
 	// Windows support `\` and `/` as its separator, Unix only use `/`
-	srcURL := strings.Replace(c.Args().Get(0), "\\", "/", -1)
-	dstURL := strings.Replace(c.Args().Get(1), "\\", "/", -1)
+	srcURL := c.Args().Get(0)
```

Comment:
My mistake, I missed a case, it should be written like this.
```suggestion
        isrcURL := c.Args().Get(0)
	dstURL := c.Args().Get(1)
	if runtime.GOOS == "windows" {
		if !strings.Contains(srcURL, "://") && isFilePath(srcURL) || strings.HasPrefix(srcURL, "file:") {
			srcURL = strings.Replace(srcURL, "\\", "/", -1)
		}
		if !strings.Contains(dstURL, "://") && isFilePath(srcURL) || strings.HasPrefix(dstURL, "file:") {
			dstURL = strings.Replace(dstURL, "\\", "/", -1)
		}
	}
```
    on windows
    1. juicefs sync file:\\\C:a\b\    s3:xxxxx\
    2. juicefs sync a\b\               s3:xxxxx\

Reply:
hi, isFilePath has checked this case `C:\a\b\c`?

Reply:
if srcURL is `file:\\\C:a\b\`, ` runtime.GOOS == "windows" && !strings.Contains(srcURL, "://") && isFilePath(srcURL) ` is false.

Reply:
Most users prefer to use the path C:a\b\ rather than to use these path `file:\\\C:a\b\`, And if we want to support this case, is it better to put these check on isFilePath?

Reply:
Yes, we can make the `isFilePath` function support `file:\\\C:a\b\`, `C:a\b\`, `a\b`.

Reply:
After discussion, we think there are only 3 legal path on windows.
1. `file:///C:/a/b/c`
2. `C:a\b\`
3. `a\b`

I'm really sorry that you repeatedly modified the code because of my problems.

Reply:
ok. Done